### PR TITLE
fixes combat carriers instantly hugging in melee

### PR DESCRIPTION
**Pull request by MacBlaze1**
## About The Pull Request

This PR ups the time for instant application of a hugger to face in melee

## Why It's Good For The Game

JPR told me to do this 

## Changelog
:cl:

sprites: thrown huggers have a different sprite from dead ones
fix: facehuggers now go idle when they drop from a xenos hand 
fix: facehuggers now go idle in gas
feature: carriers now can throw facehuggers in 1 click if they click on a tile and still have the previous way to throw them elsewise 

/:cl:

### CODE REVIEW

File: code/modules/mob/living/carbon/xenomorph/facehuggers.dm [hunk 1]
```diff
@@ -108,7 +108,7 @@
 	user.visible_message("<span class='warning'>\ [user] attempts to plant [src] on [M]'s face!</span>", \
 	"<span class='warning'>We attempt to plant [src] on [M]'s face!</span>")
 	if(M.client && !M.stat) //Delay for conscious cliented mobs, who should be resisting.
-		if(!do_after(user, 10, TRUE, M, BUSY_ICON_DANGER))
+		if(!do_after(user, 30, TRUE, M, BUSY_ICON_DANGER))
```
Comment (TiviPlus) [inline code/modules/mob/living/carbon/xenomorph/facehuggers.dm:111]:
```suggestion
		if(!do_after(user, 3 SECONDS, TRUE, M, BUSY_ICON_DANGER))
```
More readable imo

**Comment by davidchan13** [comment]
This feels entirely unnecessary. Carriers don't get any stuns or direct combat abilities, if a marine is in melee range and standing still for a second for the carrier to hug them, they deserved. Why bump them to a three second window?

**Comment by Kuro-ai** [comment]
this pr didn't work,there's still instant melee hugs happening

https://gyazo.com/43a53d2b56a7f42917b9770db0fdada6

**Comment by MacBlaze1** [comment]
> this pr didn't work,there's still instant melee hugs happening
> 
> https://gyazo.com/43a53d2b56a7f42917b9770db0fdada6

fixed it

**Comment by TiviPlus** [comment]
stale, closing

**Pull request closed without merging**